instrumentation/action_view/lib/opentelemetry/instrumentation/action_view/instrumentation.rb

```diff
@@ -19,7 +19,7 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         end
 
         compatible do
-          gem_version >= MINIMUM_VERSION
+          gem_version >= MINIMUM_VERSION && defined?(::Rails)
```

Comment:
Const checks should be done in the `present` block. 

The compatible block is an additional check to check for version compatibility.